zest/releaser/svn.py

```diff
@@ -95,10 +97,16 @@ def available_tags(self):
                 sys.exit(0)
 
         tag_info = system('svn list %s%s' % (base, tags_name))
-        if 'Could not resolve hostname' in tag_info or \
-                'Repository moved' in tag_info or 'E670008' in tag_info:
+        network_errors = ['E670008',
+                          'Unable to connect',
+                          'Could not resolve hostname',
+                          'Repository moved',
+                          'E670008']
```

Comment:
`E670008` is double in this list. Maybe the new `Unable to connect` has a different error code that you wanted to add instead?